selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/dashboard/organization/OrganizationTest.java

```diff
@@ -115,16 +117,16 @@ public void operationsWithMembers() {
     }
   }
 
-  // @Test(priority = 1)
+  @Test(priority = 1)
   public void addOrganizationWithMembers() {
-    String name = generate("orgY", 4);
+    orgName = generate("orgY", 4);
```

Comment:
Please, name it as **orgWithMemberName** to avoid confusion with **organization** field.